packages/react-components/src/trace-explorer/trace-explorer-views-widget.tsx

```diff
@@ -133,7 +133,7 @@ export class ReactAvailableViewsWidget extends React.Component<ReactAvailableVie
             const annotationCategories = await tspClient.fetchAnnotationsCategories(this._selectedExperiment?.UUID, output.id);
             const annotationCategoriesResponse = annotationCategories.getModel();
             if (annotationCategories.isOk() && annotationCategoriesResponse) {
-                const categoriesList = annotationCategoriesResponse.model.annotationCategories;
+                const categoriesList = annotationCategories ? annotationCategoriesResponse.model.annotationCategories : [];
```

Comment:
line 135 does annotationCategories.getModel(). So, there is already a null pointer problem and the check here is not needed. You need to verify it earlier.

Reply:
annotationCategories is a TspClientResponse < GenericResponse < AnnotationCategoriesModel > >
annotationCategoriesResponse is a GenericResponse < AnnotationCategoriesModel >
annotationCategoriesResponse.model is a AnnotationCategoriesModel that is nullable.

It is that last field (annotationCategoriesResponse.model) that needs a null check.

Reply:
github swallows brackets in review comments like if they were HTML tags. sigh.